packages/manager/modules/pci/src/components/project/gateways/gateways-model-selector/template.html

```diff
@@ -18,13 +18,13 @@
                 data-ng-if="gateway.blobs.technical.bandwidth.level"
             >
                 <strong
-                    data-translate="{{ 'pci_projects_project_gateways_model_selector_bandwidth'}}"
-                    data-translate-values="{ bandwidth : gateway.blobs.technical.bandwidth.level }"
+                    data-translate="{{ 'pci_projects_project_public_gateways_add_spec_' + gateway.product  + '_bandwidth'}}"
+                    data-translate-values="{ bandwidth : gateway.blobs.technical.bandwidth.level + ' ' +gateway.blobs.technical.bandwidth.maxUnit }"
                 ></strong>
             </div>
             <div class="d-flex mt-3" data-ng-if="gateway.pricings.length > 0">
                 <strong
-                    data-translate="{{ 'pci_projects_project_gateways_model_selector_catalog_price'}}"
+                    data-translate="{{ 'pci_projects_project_public_gateways_add_spec_' + gateway.product  + '_price'}}"
```

Comment:
Since we have same text for all the sizes, it has been modified to use same translation key for all the sizes. So use `pci_projects_project_gateways_model_selector_catalog_price`.